sonar-php-plugin/src/main/java/org/sonar/plugins/php/phpunit/PhpUnitCoverageResultParser.java

```diff
@@ -48,32 +50,29 @@
 import org.sonar.plugins.php.phpunit.xml.PackageNode;
 import org.sonar.plugins.php.phpunit.xml.ProjectNode;
 
-/**
- * The Class PhpUnitCoverageResultParser.
- */
 public class PhpUnitCoverageResultParser implements BatchExtension, PhpUnitParser {
 
   // Used for debugging purposes to store measure by resource
   private static final Map<Resource, Measure> MEASURES_BY_RESOURCE = new HashMap<>();
 
   private static final Logger LOG = LoggerFactory.getLogger(PhpUnitCoverageResultParser.class);
-  private final SensorContext context;
+
   private final FileSystem fileSystem;
 
   protected Metric<Integer> linesToCoverMetric = CoreMetrics.LINES_TO_COVER;
   protected Metric<Integer> uncoveredLinesMetric = CoreMetrics.UNCOVERED_LINES;
   protected Metric<String> coverageLineHitsDataMetric = CoreMetrics.COVERAGE_LINE_HITS_DATA;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this use of "COVERAGE_LINE_HITS_DATA"; it is deprecated. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)